### PR TITLE
Automate picking the OS versions

### PR DESCRIPTION
**Pull request by aptinio**

### CODE REVIEW

File: lib/bob/job/docker_checker.ex [hunk 1]
```diff
@@ -4,34 +4,50 @@ defmodule Bob.Job.DockerChecker do
...
+          # 10
+          ~r/buster-\d{8}/,
+          ~r/buster-\d{8}-slim/
+        ])
+    }
+  end
+
+  defp tags(repo, regexes) do
+    url =
+      "https://raw.githubusercontent.com/docker-library/repo-info/master/repos/#{repo}/tag-details.md"
```
Comment (ericmj) [inline lib/bob/job/docker_checker.ex:39]: I am not a fan of pulling this information from a markdown file using regex.

Maybe we can use the dockerhub API instead? We could use an uncached version of this API call https://github.com/hexpm/bob/blob/main/lib/bob/docker_hub.ex#L19C19-L19C19.

Comment (aptinio) [reply lib/bob/job/docker_checker.ex:39]: Yeah, that'll be better. I'll look into it. Thanks Eric!

**Comment by ericmj** [comment]
I don't think performance is a big concern here. This check runs every 15min and has plenty other slow parts.

Maybe we could parallelize the check for each OS using `Task.async_stream`?

**Comment by aptinio** [comment]
@ericmj, rebased and used `Task.async_stream`.

**Comment by ericmj** [comment]
Thank you @aptinio!

**Pull request merged**